pkg/blobinfocache/boltdb/boltdb.go

```diff
@@ -324,8 +336,52 @@ func (bdc *cache) appendReplacementCandidates(candidates []prioritize.CandidateW
 // If !canSubstitute, the returned candidates will match the submitted digest exactly; if canSubstitute,
 // data from previous RecordDigestUncompressedPair calls is used to also look up variants of the blob which have the same
 // uncompressed digest.
+//
+// If scope is nil, final result also contains blobs with lowest priority without locations that could be possibly be reused.
```

Comment:
That’s not what any caller is doing.

I have asked for that kind of comment on `appendReplacementCandidates`.